src/main.tsx

```diff
@@ -16,7 +17,7 @@ ReactDOM.createRoot(document.getElementById('root')!).render(
       <SpeedInsights />
       <ColorModeScript initialColorMode={theme.config.initialColorMode} />
       <QueryClientProvider client={queryClient}>
-        <App />
+        <RouterProvider router={router} />
```

Comment:
removed app.tsx as well as css, created homepage instead of this

Reply:
oh, that can break application!, thanks informing